tests/ConcurrencyTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
```

Comment:
Since we support pluggable HTTP backends, wouldn't that be more a test of Guzzle and Guzzle\Promises than of the SDK?